test/thermodynamics/entropy_of_independent_subsystems_is_sum_of_their_entropies_test.py

```diff
@@ -0,0 +1,45 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from symplyphysics import (
+    assert_equal,
+    errors,
+    units,
+    Quantity,
+)
+
+from symplyphysics.laws.thermodynamics import (
+    entropy_of_independent_subsystems_is_sum_of_their_entropies as subadditivity_law,
+)
+
+Args = namedtuple("Args", "s1 s2 s3")
+
+
+@fixture(name="test_args")
+def test_args_fixture() -> Args:
+    s1 = Quantity(1 * units.joule / units.kelvin)
+    s2 = Quantity(2 * units.joule / units.kelvin)
+    s3 = Quantity(5 * units.joule / units.kelvin)
+    return Args(s1=s1, s2=s2, s3=s3)
+
+
+def test_law(test_args: Args) -> None:
+    result = subadditivity_law.calculate_total_entropy((test_args.s1, test_args.s2))
```

Comment:
It is more common to pass vector instead tuple here: `result = subadditivity_law.calculate_total_entropy([test_args.s1, test_args.s2])`